cmd/ccboc/cmd/commands.go

```diff
@@ -364,6 +367,97 @@ func request(method, endpoint string, buffer *bytes.Buffer) ([]byte, *errorRespo
 	return body, nil, nil
 }
 
+func getCalculationPhase() error {
+	app := tview.NewApplication()
+	table := tview.NewTable().
+		SetBorders(true)
+
+	for {
+		calculationBulk, err := getCalculationBulkData()
+		if err != nil {
+			logrus.WithError(err).Fatal()
+		}
+
+		populateTable(calculationBulk, table)
+
+		app.SetInputCapture(func(event *tcell.EventKey) *tcell.EventKey {
+			if event.Key() == tcell.KeyEsc { // to exit the app with ESC
+				app.Stop()
+				os.Exit(0)
+			}
+			if event.Key() == tcell.KeyCtrlC { // to exit the app with CTRL+C
+				app.Stop()
+				os.Exit(0)
+			}
+			return nil
+		})
+
+		go func() {
+			time.Sleep(time.Second * 3) // fetch new data every 3 seconds
+			app.Stop()
+		}()
+
+		if err := app.SetRoot(table, true).SetFocus(table).Run(); err != nil {
+			logrus.WithError(err).Fatal()
+		}
+		table.Clear()
+	}
+}
+
+func getCalculationBulkData() (*bulkv1.CalculationBulk, error) {
+	args := os.Args
+	bulkID := args[len(args)-1]
+
+	body, responseError, err := request("GET", globalConfig.APIURL+"/bulk/"+bulkID, bytes.NewBuffer(nil))
+	if err != nil {
+		return nil, err
+	} else if responseError != nil {
+		logrus.WithFields(logrus.Fields{"message": responseError.Message, "status_code": responseError.StatusCode}).Fatal("errors occurred")
+	}
+
+	var response map[string]json.RawMessage
+	if err := json.Unmarshal(body, &response); err != nil {
+		return nil, err
+	}
+
+	var bulk *bulkv1.CalculationBulk
+	if err := json.Unmarshal(response["data"], &bulk); err != nil {
+		return nil, err
+	}
+
+	return bulk, nil
+}
+
+func populateTable(calculationBulk *bulkv1.CalculationBulk, table *tview.Table) {
+	rows, cols := 0, 0
+
+	for _, calc := range calculationBulk.Calculations {
+		switch calc.Phase {
+		case calculationsv1.CompletedPhase:
+			table.SetCell(rows, cols, tview.NewTableCell(fmt.Sprintf("%s:%s", calc.Name, calculationsv1.CompletedPhase)).
```

Comment:
So you can write it as:
```suggestion
			table.SetCell(rows, cols, tview.NewTableCell(fmt.Sprintf("%s:%s", name, calculationsv1.CompletedPhase)).
```